### ISSUE TITLE
The Chilling Adventures of Sabrina

### ISSUE THREAD

**Issue by Tiwas** [SickChill/SickChill#5404]
Seems to be some inconsistency here. SC lists the new episodes as s01 eps 12-20, but the scene releases lists them as s02 eps 1-9.

https://next-episode.net/chilling-adventures-of-sabrina/season-2
https://www.imdb.com/title/tt7569592/episodes?season=2

**Comment by BenjV** [comment]
SickChill is following TVDB standaards.

https://www.thetvdb.com/series/chilling-adventures-of-sabrina/seasons/1

**Comment by gramimoss** [comment]
http://thexem.de/xem/show/4636 Shows that it's been assigned the TVDB entries but Sickchill is not searching and downloading correctly.

**Comment by Tankyspanky** [comment]
Yeah i have the same problem, it keeps downloading S1 and not the new episodes

**Comment by WebSpider** [comment]
Can you try a full refresh of the series?

**Comment by Tiwas** [comment]
Full refresh didn't help me when I had the episodes, but I made changes to the scene numbering :)

**Comment by gramimoss** [comment]
I tried a Full refresh. the scene numbering is saying for example "2x3" but when doing the search I get the following.

![image](https://user-images.githubusercontent.com/5765483/56045226-c4093400-5d38-11e9-90c7-769b1ff9c4ec.png)

```
2019-04-12 15:35:31 INFO     SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Performing episode search for Chilling Adventures of Sabrina
2019-04-12 15:35:31 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Search Mode: Episode
2019-04-12 15:35:31 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Search string: Chilling Adventures of Sabrina S01E13
2019-04-12 15:35:31 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Using global proxy: 192.168.1.193:8888
2019-04-12 15:35:31 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: GET URL: https://www.torrentday.com/t.json?24=1&26=1&34=1&q=Chilling+Adventures+of+Sabrina+S01E13&2=1&7=1 [Status: 200]
2019-04-12 15:35:31 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Data returned from provider does not contain any torrents
2019-04-12 15:35:31 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Search string: Chilling.Adventures.of.Sabrina S01E13
2019-04-12 15:35:31 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Using global proxy: 192.168.1.193:8888
2019-04-12 15:35:31 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: GET URL: https://www.torrentday.com/t.json?24=1&26=1&34=1&q=Chilling.Adventures.of.Sabrina+S01E13&2=1&7=1 [Status: 200]
2019-04-12 15:35:31 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Data returned from provider does not contain any torrents
2019-04-12 15:35:31 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Search string: The Chilling Adventures of Sabrina S01E13
2019-04-12 15:35:31 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Using global proxy: 192.168.1.193:8888
2019-04-12 15:35:32 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: GET URL: https://www.torrentday.com/t.json?24=1&26=1&34=1&q=The+Chilling+Adventures+of+Sabrina+S01E13&2=1&7=1 [Status: 200]
2019-04-12 15:35:32 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Data returned from provider does not contain any torrents
2019-04-12 15:35:32 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Search string: The.Chilling.Adventures.of.Sabrina S01E13
2019-04-12 15:35:32 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Using global proxy: 192.168.1.193:8888
2019-04-12 15:35:32 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: GET URL: https://www.torrentday.com/t.json?24=1&26=1&34=1&q=The.Chilling.Adventures.of.Sabrina+S01E13&2=1&7=1 [Status: 200]
2019-04-12 15:35:32 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Data returned from provider does not contain any torrents
2019-04-12 15:35:32 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Search Mode: Episode
2019-04-12 15:35:39 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Data returned from provider does not contain any torrents

2019-04-12 15:35:39 INFO     SEARCHQUEUE-BACKLOG-338947 :: No needed episodes found during backlog search for: [Chilling Adventures of Sabrina]

```

This means that Sickchill is not working with the scene numbering and is causing the search to look for the absolute episode.

**Comment by Tiwas** [comment]
Try changing like this. Worked for me. Not sure about downloading as I downloaded them manually, but post processing worked.

![image](https://user-images.githubusercontent.com/17053402/56045984-94f7c000-5d42-11e9-9b97-662a88aa72d1.png)

**Comment by gramimoss** [comment]
So I found that I needed to enable Scene numbering and ran a full update on the tvshow. when I tried to post process the ones I downloading only one went to the right place and now I have a Season 2 table. 

![image](https://user-images.githubusercontent.com/5765483/56046484-4abf1080-5d3b-11e9-8c3f-85b8c1fc0d69.png)


Downloading still not working with the following messages.

```
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: The result The.Chilling.Adventures.of.Sabrina.S02E03.720p.WEBRip.x264-STRiFE doesn't seem to match an episode that we are currently trying to snatch, skipping it
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Adding item from search to cache: The.Chilling.Adventures.of.Sabrina.S02E03.720p.WEBRip.x264-STRiFE
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Added RSS item: [The.Chilling.Adventures.of.Sabrina.S02E03.720p.WEBRip.x264-STRiFE] to cache: [torrentday]
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Using cached parse result for: The.Chilling.Adventures.of.Sabrina.S02E03.720p.WEBRip.x264-STRiFE
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: The result The.Chilling.Adventures.of.Sabrina.S02E03.720p.WEBRip.x264-STRiFE doesn't seem to match an episode that we are currently trying to snatch, skipping it
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Adding item from search to cache: The.Chilling.Adventures.of.Sabrina.S02E03.720p.WEBRip.x264-STRiFE
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Added RSS item: [The.Chilling.Adventures.of.Sabrina.S02E03.720p.WEBRip.x264-STRiFE] to cache: [torrentday]
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Using cached parse result for: The.Chilling.Adventures.of.Sabrina.S02E03.720p.WEBRip.x264-STRiFE
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: The result The.Chilling.Adventures.of.Sabrina.S02E03.720p.WEBRip.x264-STRiFE doesn't seem to match an episode that we are currently trying to snatch, skipping it
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Adding item from search to cache: The.Chilling.Adventures.of.Sabrina.S02E03.720p.WEBRip.x264-STRiFE
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Added RSS item: [The.Chilling.Adventures.of.Sabrina.S02E03.720p.WEBRip.x264-STRiFE] to cache: [torrentday]
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Using cached parse result for: The.Chilling.Adventures.of.Sabrina.S02E03.720p.WEBRip.x264-STRiFE
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: The result The.Chilling.Adventures.of.Sabrina.S02E03.720p.WEBRip.x264-STRiFE doesn't seem to match an episode that we are currently trying to snatch, skipping it
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Adding item from search to cache: The.Chilling.Adventures.of.Sabrina.S02E03.720p.WEBRip.x264-STRiFE
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Added RSS item: [The.Chilling.Adventures.of.Sabrina.S02E03.720p.WEBRip.x264-STRiFE] to cache: [torrentday]
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Converted parsed result The.Chilling.Adventures.of.Sabrina.S02E03.720p.HEVC.x265-MeGusta into The Chilling Adventures of Sabrina - S01E14 [GROUP: MeGusta] [ABD: False] [ANIME: False] [whichReg: [u'standard']]
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Parsed The.Chilling.Adventures.of.Sabrina.S02E03.720p.HEVC.x265-MeGusta into The Chilling Adventures of Sabrina - S01E14 [GROUP: MeGusta] [ABD: False] [ANIME: False] [whichReg: [u'standard']]
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: The result The.Chilling.Adventures.of.Sabrina.S02E03.720p.HEVC.x265-MeGusta doesn't seem to match an episode that we are currently trying to snatch, skipping it
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Adding item from search to cache: The.Chilling.Adventures.of.Sabrina.S02E03.720p.HEVC.x265-MeGusta
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Added RSS item: [The.Chilling.Adventures.of.Sabrina.S02E03.720p.HEVC.x265-MeGusta] to cache: [torrentday]
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Using cached parse result for: The.Chilling.Adventures.of.Sabrina.S02E03.720p.HEVC.x265-MeGusta
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: The result The.Chilling.Adventures.of.Sabrina.S02E03.720p.HEVC.x265-MeGusta doesn't seem to match an episode that we are currently trying to snatch, skipping it
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Adding item from search to cache: The.Chilling.Adventures.of.Sabrina.S02E03.720p.HEVC.x265-MeGusta
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Added RSS item: [The.Chilling.Adventures.of.Sabrina.S02E03.720p.HEVC.x265-MeGusta] to cache: [torrentday]
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Using cached parse result for: The.Chilling.Adventures.of.Sabrina.S02E03.720p.HEVC.x265-MeGusta
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: The result The.Chilling.Adventures.of.Sabrina.S02E03.720p.HEVC.x265-MeGusta doesn't seem to match an episode that we are currently trying to snatch, skipping it
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Adding item from search to cache: The.Chilling.Adventures.of.Sabrina.S02E03.720p.HEVC.x265-MeGusta
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Added RSS item: [The.Chilling.Adventures.of.Sabrina.S02E03.720p.HEVC.x265-MeGusta] to cache: [torrentday]
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Using cached parse result for: The.Chilling.Adventures.of.Sabrina.S02E03.720p.HEVC.x265-MeGusta
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: The result The.Chilling.Adventures.of.Sabrina.S02E03.720p.HEVC.x265-MeGusta doesn't seem to match an episode that we are currently trying to snatch, skipping it
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Adding item from search to cache: The.Chilling.Adventures.of.Sabrina.S02E03.720p.HEVC.x265-MeGusta
2019-04-12 15:46:09 DEBUG    SEARCHQUEUE-BACKLOG-338947 :: [TorrentDay] :: Added RSS item: [The.Chilling.Adventures.of.Sabrina.S02E03.720p.HEVC.x265-MeGusta] to cache: [torrentday]
```

**Comment by gramimoss** [comment]
After a very long weekend of trying multiple things, I have come right. here are a few of the problems I had. 

First the Scene numbering didn't work shown in my previous comment this was fixed by editing the tv show and ticking the use Scene numbering. after that I had to do a Full update for it to start searching for the correct episode number but found it wouldn't download due to the episode number being wrong. I tried many back and forth trying to get it to download but nothing worked. 

The last resort was pulling the power on SickChill and rebooting the host completely I had tried restarts on SickChill but this never resulted in any change. after a full reboot SickChill was now working correctly and was downloading the episodes and inserting them to the correct episode number. 

So my advise to anyone that has the same problem is to set the scene numbers if they are not already, tick the box to enable scene numbering, do a full update on the tv show and lastly do a full reboot to make sure everything get setup. 

I help this helps someone.
